lib/interface/cli/commands/pipeline/run.cmd.js

```diff
@@ -126,6 +132,7 @@ const run = new Command({
             .example('codefresh run PIPELINE_ID | PIPELINE_NAME -b=master', 'Defining the source control context using a branch')
             .example('codefresh run PIPELINE_ID | PIPELINE_NAME -s=52b992e783d2f84dd0123c70ac8623b4f0f938d1', 'Defining the source control context using a commit')
             .example('codefresh run PIPELINE_ID | PIPELINE_NAME -b=master -v key1=value1 -v key2=value2', 'Setting variables through the command')
+            .example('codefresh run PIPELINE_ID | PIPELINE_NAME -b=master -v key1=value1 -v key2=value2 -e key1', 'Setting variables through the command with encrypted variables')
```

Comment:
consider channging `Setting variables through the command with encrypted variables` 
maybe to `Setting variables through the command with encrypted option`